sale_loyalty_order_line_link/tests/test_sale_loyalty_order_line_link.py

```diff
@@ -71,14 +86,20 @@ def setUpClass(cls):
             line_form.tax_id.clear()
             line_form.tax_id.add(cls.tax_2)
         cls.sale = sale_form.save()
+        cls.sale._update_programs_and_rewards()
+        cls.wizard = (
```

Comment:
If you're forcing the method in the previous line, why do you need to do it again in the wizard?

Reply:
If there are multiple programs that are applicable to the order, this method will evaluate each program and update rewards and points accordingly. Likewise in case a program has been applied and modifications are made to the sales order.

In the case that there are multiple programs applicable, the wizard will select which program to apply.

Reply:
But do you need it for the test?

Reply:
When debugging locally, the promotion is not applied automatically because it detects that there are other programs that meet the conditions to be applied, this is the standard demo data. If I delete the wizard in my database it fails because it does not apply the promotion automatically when it detects others that meet the conditions to apply. I can try to run the promotion without the wizard and test outside my database to see what happens. ....

Reply:
Ok, don't worry about it right now. It's ok if it makes the test more reliable